nx/lib/nx/random.ex

```diff
@@ -178,14 +181,21 @@ defmodule Nx.Random do
     key1 = ks[0]
     key2 = ks[1]
 
-    xs1 = xs[0] + key1
-    xs2 = xs[1] + key2
+    # slightly different representations (such as representing u32 as s64),
```

Comment:
Having the comment just for `mask32` should be fine, otherwise it can easily get out of sync :)